src/project-css/app/Helpers/TableHelper.php

```diff
@@ -0,0 +1,555 @@
+<?php
+/**
```

Comment:
This should replace your switch statement that has all the same fields being checked, I'm not sure why there is the duplicate fields, but this is exactly what i was talking about doing in the other folder making this a nice modular code.